docs/coverage/sorter.js

```diff
@@ -0,0 +1,158 @@
+var addSorting = (function () {
```

Comment:
Hmm - is this a file repeat? Not sure I totally follow what is happening here :)